libafl_qemu/Cargo.toml

```diff
@@ -17,7 +17,7 @@ all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 
 [features]
-default = ["fork", "build_libqasan", "serdeany_autoreg", "injections"]
```

Comment:
Having injections as default makes no sense if it's not supported on all archs I think

Reply:
It is

Reply:
it's not supported on i386 and hexagon

Reply:
Added support